network-api/networkapi/buyersguide/factory.py

```diff
@@ -60,29 +60,29 @@ def product_category(self, create, extracted, **kwargs):
             else:
                 return
 
-    company = Faker('company')
     blurb = Faker('sentence')
-    url = Faker('url')
-    price = LazyAttribute(lambda _: randint(49, 1500))
     camera_app = Faker('boolean')
-    meets_minimum_security_standards = Faker('boolean')
     camera_device = Faker('boolean')
-    microphone_app = Faker('boolean')
-    microphone_device = Faker('boolean')
+    company = Faker('company')
```

Comment:
most of these are me selecting all fields and going "sort alphabetically" because holy moly is it hard to find a field in a jumble of first letters >_>